docs-src/pages/UiTabs.vue

```diff
@@ -32,12 +32,10 @@
                     :key="tab.title"
                     :selected="tab.title === 'Collections'"
                     :title="tab.title"
+                    :icon="tab.icon"
 
                     v-for="tab in demoTabs"
                 >
-                    <template #icon>
```

Comment:
We should keep at least one example that uses the `icon` slot, as a test so we know it works.